src/modules/project/sagas.ts

```diff
@@ -205,12 +202,10 @@ export function* projectSaga(builder: BuilderAPI) {
         onProgress: progress => store.dispatch(setExportProgress(progress))
       })
     )
-    for (const filename of Object.keys(files)) {
-      zip.file(filename, files[filename])
-    }
 
-    const artifact: Blob = yield call([zip, 'generateAsync'], { type: 'blob' })
-    yield call(saveAs, artifact, `${sanitizedName}.zip`)
+    // download zip
+    const name = project.title.replace(/\s/g, '_')
+    yield call(downloadZip, name, files)
```

Comment:
Would you please add a test for this?

Reply:
I tried adding a `modules/project/sagas.spec.ts` like this:

```ts
import { expectSaga } from 'redux-saga-test-plan'
import * as matchers from 'redux-saga-test-plan/matchers'
import { call, select } from 'redux-saga/effects'
import { Scene } from 'modules/scene/types'
import { getData as getPools } from 'modules/pool/selectors'
import { getData as getProjects } from 'modules/project/selectors'
import { getData as getScenes } from 'modules/scene/selectors'
import { setExportProgress } from 'modules/editor/actions'
import { getName } from 'modules/profile/selectors'
import { BuilderAPI } from 'lib/api/builder'
import { projectSaga } from './sagas'
import { Project } from './types'
import { exportProjectSuccess } from './actions'
import { createFiles } from './export'
import { downloadZip } from 'lib/zip'

const builderAPI = {} as BuilderAPI

describe('when exporting a project', () => {
  let project: Project
  let projects: Record<string, Project>
  let scene: Scene
  let scenes: Record<string, Scene>
  let files: Record<string, Blob | string>

  beforeEach(() => {
    project = { id: 'aProject', title: 'Some Project', sceneId: 'aScene' } as Project
    projects = { aProject: project }
    scene = { id: 'aScene' } as Scene
    scenes = { aScene: scene }
    files = { 'model.glb': new Blob(), 'code.ts': 'console.log("hola")' }
  })

  it.skip('should download a zip with all the project files', () => {
    return expectSaga(projectSaga, builderAPI)
      .provide([
        [select(getProjects), projects],
        [select(getPools), {}],
        [select(getScenes), scenes],
        [select(getName), 'pepito'],
        [matchers.call.fn(createFiles), files],
        [call(downloadZip, 'some_project', files), undefined]
      ])
      .put(setExportProgress({ loaded: 0, total: 0 }))
      .put(exportProjectSuccess())
      .run({ silenceTimeout: true })
  })
})
```

But the `import { createFiles } from './export'` is making jest to blow up since it ends up importing babylon, and for some reason jest doesn't like it :(